tracer/test/Datadog.Trace.ClrProfiler.IntegrationTests/CI/GitParserTests.cs

```diff
@@ -115,20 +115,54 @@ public void ExtractGitDataFromFolder(TestItem testItem)
         {
             Assert.True(Directory.Exists(testItem.GitFolderPath));
```

Comment:
```suggestion
            Directory.Exists(testItem.GitFolderPath).Should().BeTrue();
```

Reply:
done in [d7bebae](https://github.com/DataDog/dd-trace-dotnet/pull/6259/commits/d7bebae2cda5caa60fd13518156b61df01d04162)